### ISSUE TITLE
Fail to use the docker/distribution as a registry

### ISSUE THREAD

**Issue by robsonpeixoto** [tsuru/tsuru#1175]
When I try to use the `docker/distribution`  as a registry I always get the error:
```
Counting objects: 6132, done.
Delta compression using up to 4 threads.
Compressing objects: 100% (2548/2548), done.
Writing objects: 100% (6132/6132), 3.54 MiB | 1.37 MiB/s, done.
Total 6132 (delta 3171), reused 6063 (delta 3115)
remote: Error in scheduler after previous errors (error in docker node "http://10.37.255.50:2375": Error: image tsuru/python: not found) trying to create container: No nodes found with one of the following metadata: pool=default
To git@my.git.host.tabajara:tsuru-dashboard.git
 ! [remote rejected] master -> master (pre-receive hook declined)
error: failed to push some refs to 'git@my.git.host.tabajara:tsuru-dashboard.git'
```

**Comment by andrewsmedina** [comment]
@robsonpeixoto thank for report. tsuru does not support docker registry 2.0 (docker/distribution) yet.

**Comment by fsouza** [comment]
As of docker/distribution#422, image removal is not supported yet, and will be available in Docker Registry 2.1 + Docker 1.7. I'm closing this issue now, so users will be able to use Docker 1.6 + Registry 2.0 with tsuru-server 0.11.

**Comment by robsonpeixoto** [comment]
Thanks @fsouza
